src/core/project/qgsproject.cpp

```diff
@@ -1936,7 +1946,13 @@ bool QgsProject::readProjectFile( const QString &filename, Qgis::ProjectReadFlag
   profile.switchTask( tr( "Resolving references" ) );
   mRootGroup->resolveReferences( this );
 
-  loadProjectFlags( doc.get() );
+  // we need to migrate old fashion designed QgsSymbolLayerReference to new ones
```

Comment:
is this related?

Reply:
No, that's an unfortunate stash/unstash error. Removed!